hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -568,15 +568,26 @@ class SheetLine(models.TransientModel):
         default=0,
     )
 
+    def _is_positive(self, value, equal=False):
+        if value > 0.0:
+            return True
+        elif value == 0.0 and equal:
```

Comment:
```suggestion
        elif float_is_zero(value) and equal:
```

Reply:
and as import:
```
from odoo.tools import float_is_zero
```
It can also be used, if needed with the digits precition or rounding if needed

Reply:
But which precision? `float_is_zero` requires a precision number